pyuvdata/uvbeam.py

```diff
@@ -978,6 +977,43 @@ def __iadd__(self, other):
         self.__add__(other, inplace=True)
         return self
 
+    def get_beam_area(self,stokes='I'):
+        """
+        Computes the integral of the beam, which has units of steradians
+        """
+        if self.beam_type is not 'power':
+            raise ValueError('beam_type must be power')
+        if self._data_normalization.value != 'peak':
+            print self._data_normalization.value
+            raise ValueError('beam must be peak normalized')
+        npix = self.data_array.shape[-1]
```

Comment:
Is this assuming a Healpix beam? If it's not healpix, there are two pixel axes that should have their length multiplied together to get the number of pixels. I'd suggest using `npix = self.Npixels` for Healpix beams and `npix = self.Naxes1 * self.Naxes2` for regularly gridded beams.

Reply:
Changed to only allow healpix.